reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -8301,6 +8301,7 @@ public final Disposable subscribe(
 	@Override
 	@SuppressWarnings("unchecked")
 	public final void subscribe(Subscriber<? super T> actual) {
+		Operators.Stacksafe stacksafe = new Operators.Stacksafe();
```

Comment:
not related to the PR, but, if we're okay with "1 additional allocation per `subscribe`", I think we can dramatically improve the TCO (and reuse it for the trampolining)